src/index.html

```diff
@@ -3,11 +3,298 @@
+        <label class="nav__toggler-label" for="nav-toggler"></label>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link" href="#gallery">
+              Галерея
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#events">
+              Виставки та події
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#lecture">
+              Відвідування
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#subscription">
+              Стати нашим другом
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Про Нас
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Магазин
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Блог
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#contacts">
+              Контакти
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="title" id="title">
+      <div class="title__logo logo"></div>
+      <p class="title__heading">худож<span class="title__heading--opacity">ній</span><br>музей</p>
+      <img
+        class="title__img"
+        src="../src/images/titlebig.jpg"
+        alt="...."
+      >
+      <img
+        class="title__img-mid"
+        src="../src/images/titlemid.jpg"
+        alt="...."
+      >
+      <div class="title__ticket ticket">
+        квитки
+        <button type="submit" class="title__button button">></button>
+      </div>
+      <p class="title__mark mark">події</p>
+      <div class="title__bottom">
+        <p class="title__heading-bottom">художнiй<br>музей</p>
+        <div class="title__ticket-bottom ticket">
+          квитки
+          <button type="button" class="title__button button">></button>
+        </div>
+      </div>
+    </section>
+    <section class="events" id="events">
+      <div class="events__title heading-section">Актуальні події</div>
+      <section class="events__element">
+        <img
+          class="events__img"
+          src="images/events/1.jpg"
+          alt="картина пейзаж"
+        >
+        <img
+          class="events__img--hidden"
+          src="images/events/1big.jpg"
+          alt="картина пейзаж"
+        >
+        <div class="events__info">
+          <div class="events__mark mark">виставка</div>
+          <div class="events__date date">26.08-29.11.2019</div>
+          <p class="events__heading">Йду і повертаюсь</p>
+          <p class="events__text">Національний Художній Музей України презентує унікальну частину колекції Градобанку - українське мистецтво другої половини 1980-1995 років.</p>
```

Comment:
```suggestion
          <p class="events__text">
             Національний Художній Музей України презентує унікальну частину колекції Градобанку - українське мистецтво другої половини 1980-1995 років.
           </p>
```

Давай всё форматировать насколько это возможно?

Reply:
Так точно!